src/App.tsx

```diff
@@ -1,156 +1,78 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import React, {
+  useEffect,
+  useState,
+} from 'react';
+import { Header } from './components/header';
+import { Main } from './components/main';
+import { Footer } from './components/footer';
+import { Notification } from './components/notification';
+import { Todo } from './types/Todo';
+import { getTodos } from './api/todos';
+
+const USER_ID = 10283;
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
+  const [todos, setTodos] = useState<Todo[] | null>(null);
+  const [error, setError] = useState<null | string>(null);
+  const [filter, setFilter] = useState<string>('all');
```

Comment:
would be better to make `Enum` for it
```suggestion
  const [filter, setFilter] = useState(...);
```